docs/Examples/Index.md

```diff
@@ -36,14 +38,48 @@ You can index fields other than the Id columns for purposes of retrieving entiti
         protected $title;
     }
 
-The above example will save 2 indices to the database that can be used to retrieve the entity. The "slug" index would
-be most useful:
+The above example will save 2 indices to the database that can be used to
+retrieve the entity. The "slug" index would be most useful:
 
     $entity = $em->retrieveByIndex('IndexedEntity', 'slug', 'my-slug');
-    
-However the second index demonstrates that an index can be a composition of fields:
+
+However the second index demonstrates that an index can be a composition of
+fields:
 
     $entity = $em->retrieveByIndex('IndexedEntity', 'id_slug', '123.my-slug');
-    
-If you modify the ID column all indices will be updated when you next persist. If you change the value of the slug, 
-then the index for the former slug is removed and a new index created. 
+
+If you modify the ID column all indices will be updated when you next persist.
+If you change the value of the slug, then the index for the former slug is
+removed and a new index created.
+
+If your a related entity implements a `__toString` method indices can be
```

Comment:
I'm pretty sure Tony was trying sooo hard to get everything grammatically correct.